court/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Court(models.Model):
+    id = models.IntegerField(primary_key=True)
+    x = models.FloatField(default=0)
+    y = models.FloatField(default=0)
+    city = models.CharField(max_length=50, blank=True)
```

Comment:
From what I know, "blank = True" means the field is considered optional.
I personally think that city field is required. Please consider changing 'blank' attribute to false.

Reply:
I'd suggesting making as fewer optional fields as possible. It will make your life easier.

Reply:
@yftacherzog do you think we should combine x and y together as a tuple?

Reply:
If I were you, I'd search for online references as I'm sure this is something others did in the past already. See an example of such discussion: https://stackoverflow.com/questions/30706799/which-model-field-to-use-in-django-to-store-longitude-and-latitude-values

Reply:
Will be updated asap. Thanks.

Reply:
Have been updated. Thank you.

Reply:
Please make that change as part of the same original commit rather than creating a new commit.